src/pyhf/schema/__init__.py

```diff
@@ -0,0 +1,32 @@
+import pathlib
```

Comment:
all brand new, to support the new api for `pyhf.schema(...)` and will support contextlib wrapping :)